emcc.py

```diff
@@ -2134,12 +2134,18 @@ def check_memory_setting(setting):
     else:
       settings.UBSAN_RUNTIME = 2
 
-  if 'leak' in sanitize:
-    settings.USE_LSAN = 1
-    default_setting('EXIT_RUNTIME', 1)
-
-    if settings.LINKABLE:
-      exit_with_error('LSan does not support dynamic linking')
+  sanitizer_mem = 0
+  if sanitize:
+    # sanitizers do at least 9 page allocs of a single page during startup.
+    sanitizer_mem += webassembly.WASM_PAGE_SIZE * 9
+    sanitizer_mem += 2097152 * 10
```

Comment:
please add a comment for this line - I think all the others are explained but this one. that constant in particular looks odd...

Or is this line a dupe of the below? `1 << 21  ===  2097152`, and 10 is almost 11..?